### ISSUE TITLE
Can't get switch_pm_text button to work

### ISSUE THREAD

**Issue by PRO2XY** [guidone/node-red-contrib-chatbot#307]
**Tl; Dr;** Can't get `switch_pm_text` button to work in inline mode. The button doesn't show up on the client. Need help in properly forming the payload. 
Reference: https://core.telegram.org/bots/api/#answerinlinequery

I am trying to get the inline mode of bot to direct my user to the bot's chat by using `switch_pm_text` and `switch_pm_parameter` string parameters in msg.payload.

The [API description](https://core.telegram.org/bots/api/#answerinlinequery) can better explain what is expected, and how to achieve it.

> `switch_pm_text`  String  Optional  
> If passed, clients will display a button with specified text that switches the user to a private chat with the bot and sends the bot a start message with the parameter `switch_pm_parameter`

So this is an extention of the `answerInlineQuery`, which requires a mandatory `results` array. The API suggests to put `switch_pm_text` and `switch_pm_parameter` strings along with the payload. However, doing this I am only able to see the content from the results array and nothing which would look like a button directing the user to the bot's page... 

I am using a function node in order to generate my response and directly send to the Telegram Sender node, as this is (probably?) not a supported feature in the Inline Query node yet.

Any help anyone? I could help port it into this library afterwards.

Here is my function node:
```
msg.payload = {};
msg.payload.type = 'inline-query-answer';
msg.payload.caching = '10';
msg.payload.content = [{
    type: 'article',
    id: 'xyz',
    title: 'title',
    input_message_content: {message_text: 'text'}
}];
msg.payload.message = 'Inline';
msg.payload.switch_pm_text = "Hello";
msg.payload.switch_pm_parameter = '/help';
return msg;
```
